erpnext_com/www/docs/user/manual/en/accounts/opening-balance/fixed_assets.md

```diff
@@ -0,0 +1,37 @@
+<!-- add-breadcrumbs -->
+# Import Fixed Assets
+
+To import all the existing fixed assets first create the asset record and then create a journal entry to update general ledger.
+
+### 1. Create Asset Records
+
+Create Asset record for each asset that your company owns which is not fully depreciated.
+
+To create a new Asset:
+
+1. Go to **Assets > Assets > New.**
+1. Enter Asset Name.
+1. Enter Item Code.
+1. Enter Location.
+1. Enter Purchase Date.
+1. Enter Gross Amount.
+1. Tick **Is Existing Asset.**
+
+<img class="screenshot" alt="Stock Asset Ledger" src="{{docs_base_url}}/assets/img/accounts/asset_opening_balance.png">
```

Comment:
this image was probably not needed

Reply:
why not...

Reply:
just a general observation, too many images can make the page a bit crowded